package.json

```diff
@@ -85,7 +85,6 @@
     "madge": "4.0.2",
     "mocha": "^6.1.4",
     "nodemon": "^2.0.16",
-    "npm-run-all": "^4.1.5",
```

Comment:
h: I think we need this package because right now this causes the build to fail in CI. 

For context, we use this packages `run-p` and `run-s` commands in a lot of our npm scripts in most of our `package.json`s

Reply:
🤦 I didn't know `run-s` and `run-p` use npm-run-all under the hood, I've always used the extensive (`npm-run-all xxx`) form previously. I'll revert this right away!